README

```diff
@@ -30,6 +30,16 @@ generate/update it automatically, you can use a dedicated file:
   $ echo "*/* $(cpuid2cpuflags)" > /etc/portage/package.use/00cpuflags
 
 
+Building
+~~~~~
+
+These are the steps necessary to build the ./cpuid2cpuflags program:
+
+  $ autoreconf -vi
+  $ ./configure -C
+  $ make -j
```

Comment:
teaching people `-j` without following integer is a stupid idea. It might work for this software but acts like a fork bomb if you do the same when building Linux™ or similar big software.

Reply:
You're right. Fixed.